velox/expression/SwitchExpr.cpp

```diff
@@ -80,7 +81,7 @@ void SwitchExpr::evalSpecialForm(
       const auto& vector = context.getField(field->index(context));
       if (vector->mayHaveNulls()) {
         decoded.decode(*vector, remaining);
-        addNulls(remaining, decoded.nulls(), context, type(), result);
+        addNulls(remaining, decoded.nulls(), context, type(), localResult);
```

Comment:
nit: we can avoid adding the nulls here, this will not initialize the vector, maybe just keep track of the nulls rows here and add it at the end when we add nulls for error rows. What do you think?

Reply:
I mean it would be the same cost